package.json

```diff
@@ -41,13 +42,13 @@
     "@react-types/numberfield": "^3.1.0",
     "@react-types/shared": "^3.8.0",
     "@sentry-internal/global-search": "^0.4.1",
-    "@sentry/integrations": "7.29.0",
-    "@sentry/node": "7.29.0",
+    "@sentry/integrations": "7.28.1",
+    "@sentry/node": "7.28.1",
```

Comment:
What happened here?

Reply:
looks like a booboo

Reply:
oops might be from a rebase I did – fixing

Reply:
done! https://github.com/getsentry/sentry/pull/43449/commits/cc9671a4e5da903f5fb90c215122e03b40ca8299